app/assets/stylesheets/content/_form_error_messages.sass

```diff
@@ -31,7 +31,8 @@
 span.errorSpan
   font-weight: bold
   width: 100%
-
+  display: contents
```

Comment:
`display: contents` is still experimental, and there are already known issues with Safari. Please have a look here for reference: https://caniuse.com/#feat=css-display-contents